### ISSUE TITLE
看不到训练进程

### ISSUE THREAD

**Issue by cheemsbaby** [ZubinGou/multi-view-prompting#3]
训练的时候卡在这一步，看不到训练进程
(mvp) root@autodl-container-a34a11a952-3cb61709:~/autodl-tmp/absa/multi-view-prompting# bash scripts/run_unified.sh
+ export CUDA_VISIBLE_DEVICES=0
+ CUDA_VISIBLE_DEVICES=0
+ cd src
+ for SEED in 5 10 15 20 25
+ K=5
+ INFER_PATH=5
+ CTRL_TOKEN=post
+ TASK=unified
+ OUT_DIR=../outputs/unified/top5_seed5
+ mkdir -p ../outputs/unified/top5_seed5
+ python main.py --data_path ../data/ --dataset seed5 --model_name_or_path t5-base --output_dir ../outputs/unified/top5_seed5 --num_train_epochs 20 --save_top_k 0 --task unified --top_k 5 --ctrl_token post --multi_path --num_path 5 --seed 5 --train_batch_size 16 --gradient_accumulation_steps 1 --learning_rate 1e-4 --lowercase --sort_label --data_ratio 1.0 --check_val_every_n_epoch 10 --agg_strategy vote --eval_batch_size 64 --constrained_decode --multi_task --do_train
Some weights of the model checkpoint at t5-base were not used when initializing MyT5ForConditionalGeneration: ['decoder.block.0.layer.1.EncDecAttention.relative_attention_bias.weight']
- This IS expected if you are initializing MyT5ForConditionalGeneration from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).
- This IS NOT expected if you are initializing MyT5ForConditionalGeneration from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).
Some weights of MyT5ForConditionalGeneration were not initialized from the model checkpoint at t5-base and are newly initialized: ['encoder.embed_tokens.weight', 'lm_head.weight', 'decoder.embed_tokens.weight']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
GPU available: True (cuda), used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
HPU available: False, using: 0 HPUs
/root/miniconda3/envs/mvp/lib/python3.8/site-packages/pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py:67: UserWarning: Starting from v1.9.0, `tensorboardX` has been removed as a dependency of the `pytorch_lightning` package, due to potential conflicts with other packages in the ML ecosystem. For this reason, `logger=True` will use `CSVLogger` as the default logger, unless the `tensorboard` or `tensorboardX` packages are found. Please `pip install lightning[extra]` or one of them to enable TensorBoard support by default
  warning_cache.warn(
You are using a CUDA device ('NVIDIA GeForce RTX 3090') that has Tensor Cores. To properly utilize them, you should set `torch.set_float32_matmul_precision('medium' | 'high')` which will trade-off precision for performance. For more details, read https://pytorch.org/docs/stable/generated/torch.set_float32_matmul_precision.html#torch.set_float32_matmul_precision
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name  | Type                         | Params
-------------------------------------------------------
0 | model | MyT5ForConditionalGeneration | 222 M
-------------------------------------------------------
222 M     Trainable params
0         Non-trainable params
222 M     Total params
891.614   Total estimated model params size (MB)

**Comment by Longer767** [comment]
The dataloader, train_dataloader, does not have many workers which may be a bo
ttleneck. Consider increasing the value of the `num_workers` argument` (try 20 which is the number of cpus on this machine) in the `DataLoader` init to improve performance.
训练的时候一直是这样看不到过程

**Comment by beeevita** [comment]
训练进程在输出目录的 log 文件里

**Comment by cheemsbaby** [comment]
但是我看不到训练多少了，是我自己的问题么，只有我自己卡在这样，每次ctrl c中止之后才能在log中看到训练多少

**Comment by ZubinGou** [comment]
You can use `tee` as in [this bash script](https://github.com/ZubinGou/multi-view-prompting/blob/4263e79439472dbbd1cec9194e125ce6e7dd4710/scripts/run_main.sh#L56).

**Comment by inksheep** [comment]
> 但是我看不到训练多少了，是我自己的问题么，只有我自己卡在这样，每次ctrl c中止之后才能在log中看到训练多少

你好，我想获得这篇论文用gdown下载的模型，自己使用gdown下载失败，想问问能发我邮箱吗，我的邮箱是775415315@qq.com。非常感谢！
